docs/docs/EntityPlayer.md

```diff
@@ -190,6 +190,10 @@ ___
 ### ClearQueueItem () {: aria-label='Functions' }
 #### void ClearQueueItem ( ) {: .copyable aria-label='Functions' }
 
+___
+### DischargeActiveItem () {: aria-label='Functions' }
```

Comment:
This vanilla function was re-hooked to add a safety check and did not actually have its functionality changed.

Therefore I don't think it should be documented here.

This also applies to FullCharge.